packages/commonwealth/client/scripts/views/pages/notification_settings2/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { GetThreadSubscriptions } from '@hicommonwealth/schemas';
+import { useCommonNavigate } from 'navigation/helpers';
+import 'pages/notification_settings/index.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import app from 'state';
+import { trpc } from 'utils/trpcClient';
+import CWPageLayout from 'views/components/component_kit/new_designs/CWPageLayout';
+import { SubEntry } from 'views/pages/notification_settings2/SubEntry';
+import { CWText } from '../../components/component_kit/cw_text';
+import { PageLoading } from '../loading';
+import './index.scss';
+
+function useThreadSubscriptions() {
```

Comment:
take a look on my another comment regarding creating hooks and proper directory structure